ci/cron/src/Main.hs

```diff
@@ -269,21 +270,13 @@ docs = do
 
 download_assets :: FilePath -> GitHubRelease -> IO ()
 download_assets tmp release = do
-    shell_ $ unlines ["bash -c '",
-        "set -euo pipefail",
-        "eval \"$(dev-env/bin/dade assist)\"",
-        "cd \"" <> tmp <> "\"",
-        "PIDS=\"\"",
-        "for ass in " <> unwords (map (show . uri) $ assets release) <> "; do",
-            "{",
-                "wget --quiet \"$ass\" &",
-            "}",
-            "PIDS=\"$PIDS $!\"",
-        "done",
-        "for pid in $PIDS; do",
-            "wait $pid",
-        "done",
-        "'"]
+    Control.Concurrent.Async.forConcurrently_ (map uri $ assets release) (\url -> do
+        shell_ $ unlines ["bash -c '",
+            "set -euo pipefail",
+            "eval \"$(dev-env/bin/dade assist)\"",
+            "cd \"" <> tmp <> "\"",
+            "wget --quiet \"" <> show url <> "\"",
```

Comment:
Seems like we could just call `wget` directly and  `-o` to avoid the `cd` but I’ll leave that up to you.

Reply:
ì'm planning to remove wget in my next PR (perhaps next-next now, if the next one is the semaphore), so I don't see much point in trying to make the Bash code nicer. I would point though that we are not in the tmpdir in terms of Haskell process cwd as far as I can tell, so the `cd` there is not superfluous as it stands.